resources/views/pages/billing/index.blade.php

```diff
@@ -308,28 +309,27 @@
                         <TD>{{$paymentMethod->expiration_month}}/{{$paymentMethod->expiration_year}}</TD>
                         <TD class="text-right">
                            @if($paymentMethod->auto == 1)
-                           {!! Form::open(['action' => ["BillingController@toggleAutoPay",$paymentMethod->id],'id' => 'deletePaymentMethodForm', 'method' => 'patch']) !!}
-                           <button class="btn btn-sm" onClick="submit(); this.disabled=true;this.innerHTML='<i class=&quot;fe fe-loader mt-2 mr-2 &quot;></i> {{utrans("billing.disabling")}}'">
+                           {!! Form::open(['action' => ["BillingController@toggleAutoPay",$paymentMethod->id],'id' => 'enablePaymentMethodForm', 'method' => 'patch']) !!}
+                           <button class="btn btn-sm btn-disable-with-msg-on-click" data-message="{{utrans("billing.disabling")}}">
                            <i class="fe fe-minus-circle mr-2"></i> {{utrans("billing.disableAuto")}}
                            </button>
                            {!! Form::close() !!}
                            @else
-                           {!! Form::open(['action' => ["BillingController@toggleAutoPay",$paymentMethod->id],'id' => 'deletePaymentMethodForm', 'method' => 'patch']) !!}
-                           <button class="btn btn-sm " onClick="submit(); this.disabled=true;this.innerHTML='<i class=&quot;fe fe-loader mt-2 mr-2&quot;></i> {{utrans("billing.enabling")}}'">
+                           {!! Form::open(['action' => ["BillingController@toggleAutoPay",$paymentMethod->id],'id' => 'enablePaymentMethodForm', 'method' => 'patch']) !!}
+                           <button class="btn btn-sm btn-disable-with-msg-on-click" data-message="{{utrans("billing.enabling")}}">
                            <i class="fe fe-check-circle mr-2"></i> {{utrans("billing.enableAuto")}}
                            </button>
                            {!! Form::close() !!}
                            @endif
                         </TD>
                         <TD class="text-right">
                            {!! Form::open(['action' => ["BillingController@deletePaymentMethod",$paymentMethod->id],'id' => 'deletePaymentMethodForm', 'method' => 'delete']) !!}
-                           <button class="btn btn-sm" onClick="submit(); this.disabled=true;this.innerHTML='<i class=&quot;fe fe-loader mt-2 mr-2 &quot;></i> {{utrans("billing.deleting")}}'">
+                           <button class="btn btn-sm btn-disable-with-msg-on-click" data-message="{{utrans("billing.deleting")}}'">
```

Comment:
Missing a opening `'` on the data message attribute.

Reply:
Yep, have dropped that extra trailing `'`.